ios/Exponent/Versioned/Core/Api/EXNotifications.m

```diff
@@ -306,6 +307,7 @@ - (instancetype)initWithExperienceId:(NSString *)experienceId kernelServiceDeleg
   UNNotificationCategory *newCategory = [UNNotificationCategory categoryWithIdentifier:[self internalIdForIdentifier:categoryId]
                                                                                actions:actionsArray
                                                                      intentIdentifiers:@[]
+                                                                     hiddenPreviewsBodyPlaceholder: previewPlaceholder
```

Comment:
If we pass in nil, does this default to the system default?

Reply:
yeah, just tested that